src/contracts/mixins/SignatureUtils.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import "@openzeppelin-upgrades/contracts/utils/cryptography/SignatureCheckerUpgradeable.sol";
+
+import "../interfaces/ISignatureUtils.sol";
+
+/// @title SignatureUtils
+/// @notice A mixin to provide EIP-712 signature validation utilities.
+/// @dev Domain name is hardcoded to "EigenLayer".
+abstract contract SignatureUtils is ISignatureUtils {
```

Comment:
can we import this in all storage contracts since it has no storage itself so inheritance order doesn't matter

Reply:
We could, but it's not needed for several core contracts.